app/extensions/brave/locales/en-US/preferences.properties

```diff
@@ -166,27 +166,27 @@ hint1=Brave will always auto-update for you, but you can check for an update on
 hint2=The File menu allows you to create a New Session Tab. Session tabs are like any other tab but they run in a different user profile.  This allows you to login to the same sites multiple times with the same browser.
 submitFeedback=Submit Feedback…
 loveToHear=We'd love to hear from you.
-startsWith=Brave starts with
-startsWithOptionLastTime=My windows / tabs from last time
+startsWith=Brave starts with:
+startsWithOptionLastTime=My windows and tabs from last time (Quit Brave with Ctrl-Q to preserve multiple windows.)
```

Comment:
This would not fit well since it is a popup menu but the "and" is a good improvement.